UPGRADE-2.0.md

```diff
@@ -0,0 +1,222 @@
+# Upgrade from 1.7 to 2.0
+
+### Client options
+
+- The `verify_ssl` option has been renamed to `ssl_verification`.
+- The `ca_cert` option has been renamed to `ssl_ca_file`.
+- The `environment` option has been renamed to `current_environment`.
+- The `http_proxy` option has been renamed to `proxy`.
+- The `processorOptions` option has been renamed to `processors_options`.
+- The `exclude` option has been renamed to `excluded_exceptions`.
+- The `send_callback` option has been renamed to `should_capture`.
+- The `name` option has been renamed to `server_name`.
+- The `project` option has been removed.
+- The `extra_data` option has been removed in favour of setting additional data
+  directly in the context.
+- The `open_timeout` option has been added to set the maximum number of seconds
+  to wait for the server connection to open.
+- The `excluded_loggers` option has been added to set the list of logger 'progname's
+  to exclude from breadcrumbs.
+- The `environments` option has been added to set the whitelist of environments
+  that will send notifications to Sentry.
+- The `serialize_all_object` option has been added to configure whether all the
+  object instances should be serialized.
+- The `context_lines` option has been added to configure the number of lines of
+  code context to capture.
+
+### Client
+
+- The constructor of the `Client` class has changed its signature. The only way
+  to set the DSN is now to set it in the options passed to the `Configuration`
+  class.
+
+  Before:
+
+  ```php
+  public function __construct($options_or_dsn = null, $options = array())
+  {
+      // ...
+  }
+  ```
+
+  After:
+
+  ```php
+  public function __construct(Configuration $config)
+  {
+      // ...
+  }
+  ```
+
+- The methods `Client::getRelease` and `Client::setRelease` have been removed.
```

Comment:
They are indeed all alike, but the advantage of this is that the file explains exactly what has changed and how to upgrade every single piece of code. As this is not the CHANGELOG file, I think it should be verbose